quisby/benchmarks/autohpl/extract.py

```diff
@@ -5,25 +5,26 @@
 
 
 def extract_autohpl_data(path, system_name):
-    
-    with open(path) as file:
-        results = []
-        file_data = file.readlines()
-
-        if len(file_data) > 1:
-            header_row = file_data[0].strip().split(":")
-            data_row = file_data[1].strip().split(":")
-           
-            data_dict = {}
-            for key, value in zip(header_row, data_row):
-                data_dict[key] = value
-
-            results = linpack_format_data(
-                results=results, system_name=system_name, gflops=data_dict["Gflops"]
-            )
-
-            if results:
-                return results
-
-        else:
-            return None
\ No newline at end of file
+
+    if path.endswith(".csv"):
```

Comment:
Should we emit a complaint if we're not handed a csv rather than just not doing anything?

Reply:
Yes I have added all these logs in next PR.